src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/VisualBasicDataFlowAnalysis.vb

```diff
@@ -82,6 +83,20 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Public Overrides ReadOnly Property DefinitelyAssignedOnEntry As ImmutableArray(Of ISymbol)
+            Get
+                If _definitelyAssignedOnEntry.IsDefault Then
+                    Dim discarded = DataFlowsIn
+                    Dim result = If(Me._context.Failed,
+                        ImmutableArray(Of ISymbol).Empty,
```

Comment:
❔ Why assume none are definitely assigned instead of all?

Reply:
Following the pattern where if anything failed, everything is empty.  It's not even clear what symbols would be returned here.  In the case of non-success, can we even report an accurate set of symbols?